README.md

```diff
@@ -1,5 +1,10 @@
 # Getting Started
 
+### Local development
+Integration tests use Testcontainers to start a PostgreSQL container. 
+To run the tests, you need to have Docker installed on your machine.
+Recommended docker application: https://rancherdesktop.io/
```

Comment:
Burde vi legge noe tips for Mac-users?
Som lenken du sendte meg f.eks.? Eller tips om admin user ish?

Reply:
Godt innspill, legger til